contracts/erc20/ERC20MintableBurnable.vy

```diff
@@ -0,0 +1,86 @@
+Transfer: event({_from: indexed(address), _to: indexed(address), _value: uint256(wei)})
```

Comment:
Could you add natspec comments like ERC721, referring to the [EIP](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md)? 🙇

Reply:
Sure, I'll do it in another PR 👍